docs/plugins/mapping.asciidoc

```diff
@@ -1,10 +1,14 @@
-[[mapper]]
-== Mapper Plugins
+[[mapping]]
```

Comment:
Does this move need to be added to the "renames" doc (I don't remember exactly what it is called)?